Whiskey/Tasks/Invoke-WhiskeyNodeNspCheck.ps1

```diff
@@ -1,9 +1,9 @@
 
 function Invoke-WhiskeyNodeNspCheck
 {
-    [Whiskey.Task("NodeNspCheck")]
-    [Whiskey.RequiresTool("Node", "NodePath", VersionParameterName='NodeVersion')]
-    [Whiskey.RequiresTool("NodeModule::nsp", "NspPath", VersionParameterName="Version")]
+    [Whiskey.Task('NodeNspCheck',Obsolete,ObsoleteMessage='The "NodeNspCheck" task is obsolete and will be removed in a future version of Whiskey. Please use the "NpmAudit" task instead. The NSP project died in September 2018 and was replaced with the `npm audit` command.')]

```

Comment:
`NpmAudit` is not a valid task. Should be pointing to the `Npm` task.

Reply:
Fixed.